tests/rules/test_anchors.py

```diff
@@ -0,0 +1,92 @@
+# Copyright (C) 2022 Sergei Mikhailov
+#
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with this program.  If not, see <http://www.gnu.org/licenses/>.
+
+from tests.common import RuleTestCase
+
+NORMAL_ANCHOR = '''---
```

Comment:
Please don't use these `NORMAL_ANCHOR`, `MISS_ANCHOR_POINTER` (I guess you've tried to reproduce the other PR), just inline the YAML code examples inside tests. All yamllint tests do this, the goal is to have tested YAML code close to the list of expected errors. Easier to read, easier to debug!

Reply:
You're right. I was following the other PR (#420 ). I agree having the yaml codes close to the expected error will probably be easier to read/debug.